packages/datadog-shimmer/src/shimmer.js

```diff
@@ -132,7 +132,7 @@ function assertFunction (target) {
 }
 
 function assertNotClass (target) {
-  if (target.toString && target.toString().startsWith('class')) {
+  if (Function.toString.call(target).startsWith('class')) {
```

Comment:
More correct here should be `Function.prototype.toString`. It works here because `Function.__proto__ === Function.prototype`, but we shouldn't count on that.

Reply:
I actually changed it to the prototype but somehow it got lost in a rebase. Will push it again.